### DISCUSSION TITLE
Bluetooth and the interface controller do not work in my 2011 Mac PowerBook with the patch

### DISCUSSION THREAD

**Discussion by ghost** [community/community#107093] [Packages]
### Select Topic Area

Question

### Body

Why does the Bluetooth and interface organizer not work in my 2011 Mac PowerBook with the patcher for all Mac systems, even the latest version, which is Sonoma?

**Comment by Cjuan4631** [comment]
**_[]()![Microsoft Privacy Statement – Microsoft privacy.PDF](https://github.com/community/community/files/14231164/Microsoft.Privacy.Statement.Microsoft.privacy.PDF)

- [ ] - @macarab # _**

**Comment by Cjuan4631** [comment]
Cjuan4631@gmail.com

**Comment by queenofcorgis** [comment]
Hi @macarab ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Apple products like [their community](https://discussions.apple.com/community/mac). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
